addons/l10n_id_efaktur_coretax/security/ir_rule.xml

```diff
@@ -2,7 +2,7 @@
 <odoo noupdate="1">
```

Comment:
Be careful that the rule is marked as no-update.
Maybe you will want to force trigger the update during the migration using `update_record_from_xml` to ensure that the rule is correctly set

Reply:
will do!